docs/migration/00b_Electronic_structure_with_v0.5.ipynb

```diff
@@ -0,0 +1,729 @@
+      "  from qiskit_nature.problems.second_quantization import ElectronicStructureProblem\n",
+      "<ipython-input-1-aea959c80ea0>:7: NatureDeprecationWarning: The qiskit_nature.problems.second_quantization.vibrational package is deprecated as of version 0.5.0 and will be removed no sooner than 3 months after the release. Instead use the qiskit_nature.second_q.problems package.\n",
+      "  from qiskit_nature.problems.second_quantization import ElectronicStructureProblem\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Fermionic Operator\n",
+      "register length=4, number terms=36\n",
+      "  -1.2563390730032498 * ( +_0 -_0 )\n",
+      "+ -0.47189600728114245 * ( +_1 -_1 )\n",
+      "+ -1.2563390730032498 * ( +_2 -_2 )\n",
+      "+ -0.47189600728114245 * ( +_3 -_3 )\n",
+      "+ -0.33785507740175813 * ( +_0 +_0 -_0 -_0 )\n",
+      "+ -0. ...\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.drivers import Molecule\n",
+    "from qiskit_nature.drivers.second_quantization import (\n",
+    "    ElectronicStructureDriverType,\n",
+    "    ElectronicStructureMoleculeDriver,\n",
+    "    PySCFDriver,\n",
+    ")\n",
+    "from qiskit_nature.problems.second_quantization import ElectronicStructureProblem\n",
+    "from qiskit_nature.transformers.second_quantization.electronic import FreezeCoreTransformer\n",
+    "from qiskit_nature.settings import settings\n",
+    "\n",
+    "settings.dict_aux_operators = True\n",
+    "\n",
+    "molecule = Molecule(\n",
+    "    geometry=[[\"H\", [0.0, 0.0, 0.0]], [\"H\", [0.0, 0.0, 0.735]]], charge=0, multiplicity=1\n",
+    ")\n",
+    "\n",
+    "driver = ElectronicStructureMoleculeDriver(\n",
+    "    molecule, basis=\"sto3g\", driver_type=ElectronicStructureDriverType.PYSCF\n",
+    ")\n",
+    "# or equivalently:\n",
+    "driver = PySCFDriver.from_molecule(molecule, basis=\"sto3g\")\n",
+    "\n",
+    "transformer = FreezeCoreTransformer()\n",
+    "\n",
+    "problem = ElectronicStructureProblem(driver, transformers=[transformer])\n",
+    "\n",
+    "# Note: at this point, `driver.run()` has NOT been called yet. We can trigger this indirectly like so:\n",
+    "second_q_ops = problem.second_q_ops()\n",
+    "\n",
+    "hamiltonian = second_q_ops[\"ElectronicEnergy\"]\n",
+    "print(hamiltonian)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "41c11c1d",
+   "metadata": {},
+   "source": [
+    "### New"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "19449c21",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Fermionic Operator\n",
+      "number spin orbitals=4, number terms=36\n",
```

Comment:
The new operator has no truncation control over the string output it seems. The previous operator had a truncated output by default.

Reply:
Yes this is on purpose. The truncation we had done there was very unnatural imo (by number of characters rather than lines for example) and I found myself disabling it more often than I ended up wanting it.

If we find it useful in the future, we could re-add that :+1: